Form-Controls/index.html

```diff
@@ -1,28 +1,73 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <title>My form exercise</title>
+  <title>Items Request</title>
   <meta name="description" content="">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="styles.css">
+  <link rel="stylesheet" href="./styles.css">
 </head>
+
 <body>
   <header>
-    <h1>Product Pick</h1>
+    <h1>Product Cart</h1>
   </header>
   <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
+    <form method="post">
+      <!-- write your html here-->
+      <!-- try writing out the requirements first-->
+      <div>
+        <label for="fname">First Name</label>
+        <input type="text" id="fname"  name="fname" placeholder="Enter your first name"  autofocus pattern="^[a-zA-Z].{1}.*" title="Name can only begin with alphabet and minimum of 2 letters" required>
+      </div>
+      <div>
+        <label for="lname">Last Name</label>
+        <input type="text" id="lname" name="lname" placeholder="Enter your last name" pattern="^[a-zA-Z].{1}.*" title="Name can only begin with alphabet and minimum of 2 letters" required>
+      </div>
```

Comment:
It's up to you, but you could join these two into one control - `name` or Full name.